### DISCUSSION TITLE
Select current line, if already selected, extend to previous line

### DISCUSSION THREAD

**Discussion by DrunkenRandomWalker** [helix-editor/helix#1100] [Q&A]
#### Describe your feature request
it would be nice to have the ability to extend the current selection to previous lines

**Comment by Omnikar** [comment]
For now, you are able to press `v`, use `k` to move up however many lines you want to select, and then `X` to extend to the line bounds.

**Comment by pickfire** [comment]
As for this specific question, I would have just `kxx`.

**Reply by Omnikar** [reply]
That requires twice as many keypresses (pressing `k` once for every line, then pressing `x` once for every line).
